src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
```

Comment:
```suggestion
  for (const { type, extraData, keysToRemove, clear } of actions) {
```

Write code for `clear` case